scripts/issues.coffee

```diff
@@ -5,12 +5,23 @@
 #   hubot issues count <user_name>@<repo_name> - get the amount of issues in a GitHub repo.
 #   hubot issues give <user_name>@<repo_name> - get a random issue from a GitHub repo.
 #
+#   issue#<user_name>/<repo_name>/<issue_number> 
+#     - displays link, title and state of issue
+#     - omitting user_name defaults to Terasology as user
+#     - omitting both user_name and repo_name defaults to MovingBlocks/Terasology repo
+#     
+#     - e.g. "Issue #1" is MovingBlocks/Terasology's #1 issue
+#            "Issue #Cooking/1" is Terasology/Cooking's #1 issue (as I think we talk more about modules, which are under the Terasology user)
+#            "Issue #MovingBlocks/GooeyJr/1" is MovingBlocks/GooeyJr's #1 issue (if we need to get an issue from a specific repo)
+#     - Also, you can shorten Issue to Iss. 
```

Comment:
Can you update your documentation using the guidelines at https://github.com/hubotio/hubot/blob/master/docs/scripting.md#documenting-scripts ?

The descriptions and the command should be on one single line to avoid messing the `help` command up. You can remove the point about shortening Issue to Iss since as suggested below we can remove it altogether. 

To shorten to rest of it, how about listing 2 different commands as follows:
- `#<user_name>/<repo_name>/<number> - retrieves issue or PR #<number> from <user_name>/<repo_name>`
- `#<number> - retrieves issue or PR #<number> from MovingBlocks/Terasology`

I think these descriptions are clear enough that we no longer need the examples.